### ISSUE TITLE
`tt cluster publish` usage is confusing and dangerous

### ISSUE THREAD

**Issue by psergee** [tarantool/tt#877]
It is unclear from `cluster publish <app> <new.yaml>` command name that it replaces a `config.yaml` in the application directory. Need more details in the help.
This command does not ask for confirmation, and a user may accidentally overwrite existing configuration (expecting ETCD pubishing, for example)
This command replaces existing configuration, but only if it is valid app configuation. Or even if the config valid, but unavailable: etcd is stopped, for example. This command should not require valid application to replace its config.

**Comment by 0x501D** [comment]
This command, unexpectedly for me, erased the replicaset config for me, and for a long time I could not understand what was happening.

**Comment by psergee** [comment]
> This command, unexpectedly for me, erased the replicaset config for me, and for a long time I could not understand what was happening. I assumed that it read etcd parameters from the main config and sent remote.yml according to them.

Yeah... It silently and unexpectedly replaced my local configuration too. No log. No confirmation. Nothing.
